testER/src/App.jsx

```diff
@@ -1,26 +1,30 @@
-import React, { useEffect, useState } from 'react'
+import React, { createContext, useEffect, useState } from 'react'
 import ReactGA from 'react-ga'
 import mapboxgl from 'mapbox-gl'
-import ReactDOM from 'react-dom'
-import SubjectPopup from './components/Popup'
-import Legend from './components/Legend.jsx'
+import SubjectPopupContent from './components/SubjectPopupContent'
+import Popup from './components/Popup'
+import Legend from './components/Legend'
 
 import './App.css'
 import 'mapbox-gl/dist/mapbox-gl.css'
 
 // instantiate the Map
 mapboxgl.accessToken = 'pk.eyJ1IjoidmpvZWxtIiwiYSI6ImNra2hiZXNpMzA1bTcybnA3OXlycnN2ZjcifQ.gH6Nls61WTMVutUH57jMJQ' // development token
-let GlobalMap
+
 let config
 const keymap = {} // alt, r
 
+window.GlobalMap = null
```

Comment:
This just attaches `GlobalMap` to the window for access in other components. It's not the right approach but it follows the pattern set. 

You preferably want to be capturing this in state, and sharing it via the Context API or some other means.